src/index.html

```diff
@@ -2,9 +2,14 @@
 <html lang = "en">
 <head>
     <meta charset="UTF-8">
-    <title>ToDo List</title>
+    <meta
+      name="viewport"
+      content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no"
+    >
+    <title>To Do List</title>
 </head>
 <body>
-    <div id="app"</div>
+    <div id="app"></div>
 </body>
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
with this explicit import, you might not need the `roboto` stuff in the `package.json`...and actually, I was reading somewhere that most browsers already have that by default (haven't messed around with it yet)...but we can specify it in the `@fontface` in the `body` selector for the CSS rules.

...also, just found this, which seems quite sensible:
https://stackoverflow.com/questions/18178867/how-can-i-use-googles-roboto-font-on-a-website